.github/workflows/foreman.yml

```diff
@@ -123,7 +123,18 @@ jobs:
           bundle exec rake db:create
           bundle exec rake db:migrate
       - name: Run rake ${{ matrix.task }}
-        run: bundle exec rake ${{ matrix.task }}
+        run: ${{ contains(matrix.task, 'integration') && 'APIPIE_RECORD=examples' || '' }} bundle exec rake ${{ matrix.task }}
+      - name: Build apidoc
+        run: bundle exec rake apipie:cache
```

Comment:
I guess instead of calling it manually here, we could extend the task from `webpack:compile test:integration` to `webpack:compile test:integration apipie:cache`?